server/routes/blog.py

```
@@ -51,6 +57,216 @@ def get(self, blog_id):
+                ).start()
+
+            return restful_response(
+                status="success",
+                message="Blog created successfully",
+                data=new_blog.to_dict(),
+                status_code=201,
+            )
+        except ValueError as e:
+            db.session.rollback()
+            return restful_response(status="error", message=str(e), status_code=400)
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)